src/runtime/core/CMakeLists.txt

```diff
@@ -120,8 +120,8 @@ target_link_libraries(
   -L${ROCM_LIBRARIES_DIR}
   -L${CMAKE_INSTALL_PREFIX}/lib -lamd_hostcall -Wl,-rpath,${CMAKE_INSTALL_PREFIX}/lib
   -Wl,--enable-new-dtags
-  -Wl,-rpath,\$ORIGIN/../../hsa/lib
-  -Wl,-rpath,\$ORIGIN/../../lib
+  -Wl,-rpath,\$ORIGIN/../hsa/lib
+  -Wl,-rpath,\$ORIGIN/../lib
```

Comment:
Same comment as above.
It would be good to clarify the relative hierarchies of AOMP and ROCm bits.